### PR TITLE
update ci versions

### PR DESCRIPTION
**Pull request by BCerki**
This PR:
- address ref #64 
- updates CI node versions to 16

### CODE REVIEW

File: .secrets.baseline
Comment (wenzowski) [inline .secrets.baseline]: No need to include this in the patch, I think

File: .github/workflows/ci.yml
Comment (wenzowski) [inline .github/workflows/ci.yml:15]: This looks like continuation of #56 ...so I'd call the fact that we were able to bump the node version there and miss the CI configuration tech debt. Let's not hold back this PR but let's figure out how to ensure when we bump node versions in the future we get everything in one go–perhaps we should be looking at how other projects use a `.tool-versions` file as a source of truth that gets consumed by other systems, like docker, ci, npm, etc.

**Review by wenzowski — DISMISSED** [review]
Crucial!

File: .secrets.baseline
Comment (BCerki) [reply .secrets.baseline]: @wenzowski, is there a way to un-include things like this without changing the commits and un-approving the PR?

Comment (wenzowski) [reply .secrets.baseline]: I believe our approval settings are currently to [dismiss stale reviews](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-pull-request-reviews-before-merging) every time there's a change in the commit manifest of the PR

**Review by wenzowski — APPROVED** [review]
Let's ship & record tech debt as ticket

**Comment by BCerki** [comment]
> Let's ship & record tech debt as ticket

Here: https://github.com/button-inc/digital_marketplace/issues/67

**Pull request merged**
